### PR TITLE
loadable_apps/binary_update: Change a scenario of update aging test

### PR DESCRIPTION
**Pull request by jeongarmy**
Change update aging test to make micom update instead of wifi.

### CODE REVIEW

File: loadable_apps/loadable_sample/wifiapp/binary_update.c [hunk 1]
```diff
@@ -669,6 +644,27 @@ void binary_update_test(void)
...
+	int ret;
+	binary_update_info_t bin_info;
+
+	while (1) {
+		ret = binary_update_getinfo(APP_NAME, &bin_info);
+		if (ret != OK) {
+			return;
+		}
+
+		/* Copy current binary with a higher version and valid crc. */
```
Comment (jeongchanKim) [inline loadable_apps/loadable_sample/wifiapp/binary_update.c:656]: `current binary` is wifiapp. Please change `current binary` to `current micom binary` or something.

File: loadable_apps/loadable_sample/wifiapp/binary_update.c [hunk 2]
```diff
@@ -669,6 +644,27 @@ void binary_update_test(void)
...
+		ret = binary_update_download_binary(&bin_info, DOWNLOAD_VALID_BIN);
+		if (ret != OK) {
+			return;
+		}
+
+		sleep(5);
+
+		ret = binary_update_reload(APP_NAME);
+		if (ret != OK) {
```
Comment (jeongchanKim) [inline loadable_apps/loadable_sample/wifiapp/binary_update.c:665]: Is it okay not to remove the downloaded binary? If not, it can consume the flash.

Comment (jeongarmy) [reply loadable_apps/loadable_sample/wifiapp/binary_update.c:665]: Right. The new file needs to be removed in this case. I will modify it

File: loadable_apps/loadable_sample/wifiapp/binary_update.c (hunk 1 above)
Comment (jeongarmy) [reply loadable_apps/loadable_sample/wifiapp/binary_update.c:656]: ok. I will modify it.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**